api/python/t4/api.py

```diff
@@ -449,3 +450,121 @@ def config(*autoconfig_url, **config_values):
         write_yaml(local_config, CONFIG_PATH, keep_backup=True)
 
     return HeliumConfig(CONFIG_PATH, local_config)
+
+def create_role(name, arn):
+    """
+    Create a new role in your registry. Admins only.
+
+    Required Parameters:
+        name(string): name of role to create
+        arn(string): ARN of IAM role to associate with the Quilt role you are creating
+    """
+    session = get_session()
+    response = session.post(
+        "{url}/api/roles".format(
+            url=get_registry_url()
+            ),
+            data=json.dumps({
+                'name': name,
+                'arn': arn
+            })
+        )
+    if not response.ok:
+        message = response.json()['message']
```

Comment:
You could also get a 500, which might not be valid JSON. Also, `session.post` could raise an exception for connection errors, etc.

We probably want to take the error handling from Quilt2. Though could punt on that for now, I guess

Reply:
My intention is to use the error handling from Quilt2. I added this block by mistake, removing it now.

Reply:
If that doesn't make it into this PR, can you throw a TODO in?